bin/stalePullRequest.js

```diff
@@ -1,24 +1,35 @@
 'use strict';
 
 var Cesium = require('cesium');
+var handlebars = require('handlebars');
 var Promise = require('bluebird');
 var requestPromise = require('request-promise');
 
 var defined = Cesium.defined;
 
-var dateLog = require('../lib/dateLog');
 var checkStatus = require('../lib/checkStatus');
+var dateLog = require('../lib/dateLog');
 var Settings = require('../lib/Settings');
 
 module.exports = stalePullRequest;
 
+var firstMessage = 'Thank you for the pull request!\n\n' +
```

Comment:
I'm confused as to why these are still here.  I would expect the defaults to be in Settings.